py/desitarget/mock/mockmaker.py

```diff
@@ -2124,10 +2127,13 @@ def readmock(self, mockfile=None, healpixels=None, nside=None,
 
         """
         from astropy import cosmology
-
         try:
             from simqso.sqbase import ContinuumKCorr
-            from simqso.sqmodels import BOSS_DR9_PLEpivot
+            if sqmodel is 'default':
+                from simqso.sqmodels import BOSS_DR9_PLEpivot as model_PLEpivot
+            else:
+                from desisim.scripts.lya_simqso_model import model_PLEpivot
+
             from simqso.sqgrids import generateQlfPoints
```

Comment:
https://github.com/desihub/desisim/pull/523 will need to be merged first, of course.